src/components/applicantResponse.js

```diff
@@ -1,36 +1,79 @@
 // this is the third sidebar for the scoring page
 
-import React from 'react'
+import React, { useState } from 'react'
 import styled from 'styled-components'
 import ResponseInput from './responseInput'
+import { COLOR } from '../constants'
+import { render } from '@testing-library/react';
 
 
 const Main = styled.div`
     padding: 20px;
     border: 1px solid gray;
     margin-right: 30px;
-    width: 40%;
+    text-align: left;
 `;
 
-const Tabs = styled.div`
-    display: flex;
-    flex-direction: row;
 
+const Tab = styled.div`
+    margin-right: 20px;
+    :hover {
+        color: ${COLOR.BLUE_TEXT};
+        cursor: pointer;
+    }
 `;
 
 export default function ApplicantResponse() {
 
-    const tabs = ["Overview", "Resume", "Comments"] 
+    const tabs = ["Overview", "Resume", "Comments"]
+
+    const [activeTab, setActiveTab] = useState('Overview')
+
+  
+
 
     return(
         <Main>
-            <Tabs>
-                <h3>Overview</h3>
-                <h3>Resume</h3>
-                <h3>Comments</h3>
-            </Tabs>
-            <ResponseInput label="Is this your first hackathon?" response="yes" />
-            <ResponseInput label="GitHub/GitLab/BitBucket" reponse="https://github.com/yungalyx" />
+            <div style={{ "display": "flex", "flexDirection": "row"}}>
+                <Tab onClick={()=>(setActiveTab('Overview'))}> Overview </Tab>
+                <Tab onClick={()=>(setActiveTab('Resume'))}> Resume </Tab>
+                <Tab onClick={()=>(setActiveTab('Comments'))}> Comments </Tab>
+            </div>
+    
+            <div>
+                Current tab is {activeTab}
+            </div>
+
+            {activeTab === 'Overview' ?
+                <OverviewTab> </OverviewTab> :
+                activeTab === 'Resume' ?
```

Comment:
use the tabs constant from line 28

maybe change the constant tabs to 
const OVERVIEW = "Overview"
const RESUME = "RESUME"
const COMMENTS = "Comments"

instead of storing it in an array